.pfnci/linux/main-flexci.sh

```diff
@@ -37,8 +37,14 @@ gcloud auth configure-docker
 
 echo "Starting: "${TARGET}""
 echo "****************************************************************************************************"
-CACHE_DIR=/tmp/cupy_cache PULL_REQUEST="${pull_req}" "$(dirname ${0})/run.sh" "${TARGET}" cache_get build test 2>&1 | tee "${LOG_FILE}"
+
+STAGES="cache_get build test"
+if [[ "${TARGET}" == "benchmark" ]]; then
+    STAGES="cache_get build benchmark"
+fi
+BENCHMARK_DIR=/tmp/benchmark CACHE_DIR=/tmp/cupy_cache PULL_REQUEST="${pull_req}" "$(dirname ${0})/run.sh" "${TARGET}" "${STAGES}" 2>&1 | tee "${LOG_FILE}"
```

Comment:
Need to upload `BENCHMARK_DIR`?

Reply:
No, we won't be uploading but running both branches in the PR I am removing this references.

We may want to upload stuff in the future maybe

Reply:
Ah understood, so we can check the result in the console output!